.rhcicd/clowdapp-backend.yaml

```diff
@@ -343,3 +357,9 @@ parameters:
   value: "40"
 - name: LIVENESS_INITIAL_DELAY
   value: "40"
+- name: OIDC_CLIENT_ENABLED
+  description: If true, the OIDC client will be enabled.
+  value: "true"
```

Comment:
Shouldn't it be false by default? Event if not used in runtime code because of unleash FF, we could avoid to init this on startup in env who can't support it yet?